events/2022.03.Online/TD/Fujitsu/TDs/fjsensor.td.jsonld

```diff
@@ -0,0 +1,57 @@
+{
+  "@context": [
+    "https://www.w3.org/2019/wot/td/v1",
+    "https://www.w3.org/2022/wot/td/v1.1"
+  ],
+  "@type": "Thing",
+  "title": "Fujitsu-sensor",
+  "id": "urn:com:fujitsu:sensor01",
+  "description": "sensors with wifi interface",
+  "version": "1.0.0",
```

Comment:
This is incorrect, it should be an object: see https://w3c.github.io/wot-thing-description/#versioninfo

Reply:
Thanks for your check. Fixed it.